code/modules/mob/living/carbon/xenomorph/castes/lesser_drone.dm

```diff
@@ -16,9 +16,10 @@
 	can_be_revived = FALSE
 
 	build_time_mult = BUILD_TIME_MULT_LESSER_DRONE
+	behavior_delegate_type = /datum/behavior_delegate/lesser_drone_base
 
 	caste_desc = "A builder of hives."
-	can_hold_facehuggers = 1
+	can_hold_facehuggers = 0
```

Comment:
```suggestion
	can_hold_facehuggers = TRUE
```
Its fine they can hold huggers. Make it instead that huggers transfer their time on the ground to their in hand variant so you cannot reset the timer and they expire in hands. Carriers however should reset the timer in their hand and not expire in their hands.

Reply:
Unresolved. Likely just need a check for carrier in the code that was removed to make it only occur for carriers.

Reply:
Should be fixed
Carriers now prevent them from losing hp while in their hands